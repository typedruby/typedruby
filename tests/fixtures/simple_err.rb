# @typedruby

def add1(Integer x) => Integer
  x + 1
end

def use_add1 => Integer
  add1("0")
end
