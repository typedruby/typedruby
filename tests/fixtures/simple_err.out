error: Could not match types:

        @ __ROOT__/tests/fixtures/simple_err.rb:1
      1 |  def add1(Integer x) => Integer
                    ^^^^^^^ Integer, with:
        @ __ROOT__/tests/fixtures/simple_err.rb:6
      6 |    add1("0")
                  ^^^ String
        @ __ROOT__/tests/fixtures/simple_err.rb:6
      6 |    add1("0")
             ^^^^^^^^^ in this expression
